CITATION.cff

```diff
@@ -0,0 +1,16 @@
+type: software
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Arya"
+  given-names: "Abhishek"
+- family-names: "Brown"
+  given-names: "Caleb"
+- family-names: "Pike"
+  given-names: "Rob"
+- name: "The Open Source Security Foundation" 
+title: "Open Source Project Criticality Score"
+date-released: 2020-12-09
```

Comment:
I'm not that familiar with cff - should this be the first release date, or the date that corresponds to the release of the version specified below (i.e. 2.0.2).

Reply:
Ya, I had the same question. After a bit more digging it looks like best practice is to use release date (similar to using 5th edition of a book). There some background info here: https://www.software.ac.uk/how-cite-software.

I updated the date to match.